lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :basket, :menu
+
```

Comment:
What happens if you want to change the menu?  Or create several Takeaways with different menus?